### PR TITLE
Rename various Presto references to Trino

### PR DESCRIPTION
**Pull request by phd3**
Another pass after https://github.com/trinodb/trino/pull/6484/commits/f6422d04663ab011d3ddb831ed16dec02659c47e. The renamings in this PR are independent and inconsequential.

### CODE REVIEW

**Review by findepi — APPROVED** [review]
(skimmed)

File: plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
 import static java.util.stream.Collectors.joining;
 
 // holds caches and mappings
-// presto converts the dataset and table names to lower case, while BigQuery is case sensitive
+// trino converts the dataset and table names to lower case, while BigQuery is case sensitive
```
Comment (electrum) [inline plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java:47]: Capitalize

File: plugin/trino-jmx/src/main/java/io/trino/plugin/jmx/JmxMetadata.java [hunk 2]
```diff
@@ -223,7 +223,7 @@ else if (HISTORY_SCHEMA_NAME.equals(schema)) {
     {
         ImmutableList.Builder<SchemaTableName> tableNames = ImmutableList.builder();
         for (ObjectName objectName : mbeanServer.queryNames(WILDCARD, null)) {
-            // todo remove lower case when presto supports mixed case names
+            // todo remove lower case when trino supports mixed case names
```
Comment (electrum) [inline plugin/trino-jmx/src/main/java/io/trino/plugin/jmx/JmxMetadata.java:226]: Capitalize

**Review by electrum — COMMENTED** [review]
Nice work! Thanks for doing this.

**Review by electrum — APPROVED** [review]

**Pull request merged**